src/database-controller/src/common/framework.js

```diff
@@ -23,6 +23,21 @@ const mockFrameworkStatus = () => {
   };
 };
 
+const mockFailedFrameworkStatus = () => {
+  return {
```

Comment:
We can only produce event and does not fail job in first stage.

Here the mocked failed state is not completed, such as miss the
attemptStatus's id, startTime, completionTime, completionStatus, etc.
status's startTime, completionTime, transitionTime, etc.

Reply:
You can check other failed jobs to figure out all missing field here.

---
In reply to: [570746620](https://github.com/microsoft/pai/pull/5284#discussion_r570746620) [](ancestors = 570746620)